src/github.com/airvantage/moxy/server.go

```diff
@@ -4,13 +4,14 @@ import (
 	"bufio"
 	"bytes"
 	"encoding/hex"
-	"git.eclipse.org/gitroot/paho/org.eclipse.paho.mqtt.golang.git/packets"
 	"io"
 	"io/ioutil"
 	"log"
 	"net"
 	"os"
 	"strconv"
+
```

Comment:
Nope, atom's go mode uses [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) ; I'll change it back to `gofmt`.